mail_activity_board/models/mail_activity.py

```diff
@@ -81,3 +81,25 @@ def _find_allowed_doc_ids(self, model_ids):
                 continue
             allowed_ids |= self._find_allowed_model_wise(doc_model, doc_dict)
         return allowed_ids
+
+    @api.model
+    def _search(
+        self,
+        args,
+        offset=0,
+        limit=None,
+        order=None,
+        count=False,
+        access_rights_uid=None,
+    ):
+        result = super()._search(
```

Comment:
Would be nice to have your chrystal clear explanation from the PR here as a docstring!